### PR TITLE
add loading spinner and text

### PR DESCRIPTION
**Pull request by xiwcx**
https://jira.nationalgeographic.com/browse/SYTO-258

**Testing Instructions**:

1. checkout branch
- grunt serve
- visit http://localhost:3000/blocks.html#loadingAnimation

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -9,6 +9,9 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.scss]
+indent_size = 2
```
Comment (pedromagnus) [inline .editorconfig]: Some files have 4 spaces (in this PR).
Also, 2 is enough? I personally prefer 4 for easier to look/read it. But as it's standarized  I go with any option.

File: app/src/styles/blocks/_loading.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+*/
+
+$mt_loadingsize-sm: $mt_unit;
+$mt_loadingthickness-sm: $mt_unit / 3;
+$mt_loadingsize-md: $mt_loadingsize-sm * 2;
+$mt_loadingthickness-md: $mt_loadingthickness-sm * 2;
+$mt_loadingsize-lg: $mt_loadingsize-sm * 3;
+$mt_loadingthickness-lg: $mt_loadingthickness-sm * 3;
+
+$mt_loadinganimationduration: .6s;
```
Comment (pedromagnus) [inline app/src/styles/blocks/_loading.scss]: A stupid sugestion: what about a little slower? Like `1s` or a little more.

**Comment by pedromagnus** [comment]
Looks good!

File: app/src/styles/blocks/_loading.scss (hunk 2 above)
Comment (xiwcx) [inline app/src/styles/blocks/_loading.scss]: Not stupid, did you already try it out and like the result? I'll try it out too.

**Pull request merged**